tests/acceptance/29_simulate_mode/files_promises.cf.sub

```diff
@@ -0,0 +1,31 @@
+bundle common simulation_files_promises
+{
+  vars:
+    "inputs" slist => { "$(this.promise_dirname)/../default.cf.sub" };
+}
+
+body common control
+{
+  inputs => { "@(simulation_files_promises.inputs)" };
+}
+
+bundle agent main
+{
+  files:
+    "$(G.testroot)$(const.dirsep)create-true"
```

Comment:
I'll just keep adding tests here and have one large expected file to test against.